src/features/signup/components/faqList.tsx

```diff
@@ -0,0 +1,35 @@
+import { FAQItem } from "./faqItem";
+
+export const FAQList = (): JSX.Element => (
+  <div className="mt-28 flex flex-col items-center justify-center">
+    <h1 className="mb-8">FAQ</h1>
+
+    <FAQItem
+      description="This found is focused on supporting Mexican projects working on reducing the gender gap, collaborating with local communities, and ReFi."
```

Comment:
typo: "round"

Reply:
Let's remove mentions to EthMexico? "This round is focused on supporting public goods projects which made an impact on the Ethereum ecosystem" (to be more general)